### ISSUE TITLE
No typings in 1.0.1 npm release

### ISSUE THREAD

**Issue by adriengibrat** [cozmo/jsQR#45]
Hi, Thanks for the good work on jsQR 1.x, The API is now clear and simple ;)

I'm using jsQR 1.0.1 in a typescript project and don't get why the [jsqr 1.0.1 npm package](https://registry.npmjs.org/jsqr/-/jsqr-1.0.1.tgz) does not contains [the typings](https://github.com/cozmo/jsQR/tree/cdc98b54921f50f3508869c1490b12803816a256/dist).

Could you publish a 1.0.2 with the typings included (and may be deprecate the 1.0.1 with a message inviting to upgrade to get typings), something like:

```
npm version patch -m "Upgrade to %s to include typings"
npm publish
npm deprecate jsqr@1.0.1 "Upgrade to >1.0.1 get typings"
git push
git push --tags
```

thanks ;)

**Comment by cozmo** [comment]
wtf this is very odd. I want to investigate for a second to make sure there's not a systemic issue in the publishing setup, but then will publish and deprecate. Thanks for the report.

**Comment by cozmo** [comment]
Hmm, so it would appear that types are also not present in `v1.0.0` which is at least good in that it means there wasn't some one off oddity with 1.0.1 but still need to figure out why it's happening.

**Comment by adriengibrat** [comment]
The `.npmignore` seems OK to me but:
- you could remove the `*` and add an actual black list
- you could also try to add a [`files` entry](https://docs.npmjs.com/files/package.json#files) in package.json

To check what will be published, [run `npm pack`](https://docs.npmjs.com/cli/pack)

**Comment by cozmo** [comment]
Just published `v1.0.2` which has the types. #46 has the details of the bug if you're curious. Thanks for the report, sorry about the issue.

**Comment by adriengibrat** [comment]
:+1:
